Stripe/STPPaymentHandlerActionParams.m

```diff
@@ -0,0 +1,134 @@
+//
+//  STPPaymentHandlerActionParams.m
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 6/28/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentHandlerActionParams.h"
+
+#import "STPAPIClient.h"
+#import <Stripe3DS2/Stripe3DS2.h>
+#import "STPPaymentIntent.h"
+#import "STPSetupIntent.h"
+#import "STPThreeDSCustomizationSettings.h"
+#import "STPThreeDSCustomization+Private.h"
+
+@implementation STPPaymentHandlerPaymentIntentActionParams
+{
+    BOOL _serviceInitialized;
+}
+
+@synthesize threeDS2Service = _threeDS2Service;
+
+- (instancetype)initWithAPIClient:(STPAPIClient *)apiClient
+            authenticationContext:(nullable id<STPAuthenticationContext>)authenticationContext
+     threeDSCustomizationSettings:(STPThreeDSCustomizationSettings *)threeDSCustomizationSettings
+                    paymentIntent:(STPPaymentIntent *)paymentIntent
+                       completion:(STPPaymentHandlerActionPaymentIntentCompletionBlock)completion {
+    self = [super init];
+    if (self) {
+        _apiClient = apiClient;
+        _authenticationContext = authenticationContext;
+        _threeDSCustomizationSettings = threeDSCustomizationSettings;
+        _paymentIntent = paymentIntent;
+        _paymentIntentCompletion = [completion copy];
+    }
+    
+    return self;
+}
+
+- (nullable STDSThreeDS2Service *)threeDS2Service {
+    if (!_serviceInitialized) {
+        _serviceInitialized = YES;
+        _threeDS2Service = [[STDSThreeDS2Service alloc] init];
+        @try {
+            STDSConfigParameters *configParams = [[STDSConfigParameters alloc] initWithStandardParameters];
+            [configParams addParameterNamed:@"kInternalStripeTestingConfigParam" withValue:@"Y"];
```

Comment:
we really need a followup to remove this before shipping...

Reply:
https://jira.corp.stripe.com/browse/IOS-1285